compiler/rustc_query_system/src/query/job.rs

```diff
@@ -619,30 +620,48 @@ pub fn print_query_stack<Qcx: QueryContext>(
     mut current_query: Option<QueryJobId>,
     handler: &Handler,
     num_frames: Option<usize>,
+    mut file: Option<std::fs::File>,
 ) -> usize {
     // Be careful relying on global state here: this code is called from
     // a panic hook, which means that the global `Handler` may be in a weird
     // state if it was responsible for triggering the panic.
     let mut i = 0;
+    let mut j = 0;
```

Comment:
what do these numbers mean? Please name them more helpfully and document the return value on the function.